isy-web-lib/src/main/java/de/bund/bva/isyfact/common/web/tempwebresource/controller/ResourceController.java

```diff
@@ -124,9 +114,9 @@ protected TempWebResourceRo ladeTempWebResource(HttpServletRequest request) {
 
         TempWebResourceRo webResource = null;
         try {
-            String benutzerkennung =
-                this.aufrufKontextVerwalter.getAufrufKontext().getDurchfuehrenderBenutzerKennung();
-            String bhknz = this.aufrufKontextVerwalter.getAufrufKontext().getDurchfuehrendeBehoerde();
+            String benutzerkennung = (String) berechtigungsmanager.getTokenAttribute("preferred_username");
```

Comment:
To avoid the issue, the Berechtigungsmanager has now been removed and the token attributes are read directly from the SecurityContext. The code is similar to IsySecurityTokenUtil
https://github.com/IsyFact/isyfact-standards/blob/feature/ISY-305_Token-Util_zum_Auslesen_aus_dem_Bearer_Token/isy-security/src/main/java/de/bund/bva/isyfact/security/oauth2/util/IsySecurityTokenUtil.java
, cf. there for more context.